README.md

```diff
@@ -21,7 +27,35 @@ helm install gremlin gremlin/gremlin \
     --set gremlin.secret.teamSecret=YOUR-TEAM-SECRET
 ```
 
-For more detailed instructions, see the chart's documentation [here](https://github.com/gremlin/helm/blob/master/gremlin/README.md).
+For more detailed instructions, see the chart's
+documentation [here](https://github.com/gremlin/helm/blob/master/gremlin/README.md).
+
+### Gremlin Integration
+
+#### Secret Auth
+
+```shell
+helm install gremlin gremlin/gremlin-integrations \
```

Comment:
note: the way you've constructed this command it should overlap with the existing gremlin installation thus preventing both from working on the same cluster.  Is that intentional?

Reply:
That isn't intentional. It was a mistake from my side.